htdocs/takepos/admin/bar.php

```diff
@@ -155,9 +155,9 @@
         print '<tr class="oddeven value"><td>';
 		print $langs->trans("Table")." ".$row['label'];
 		print '<td>';
-		print "<a target='_blank' href='".$urlwithroot."/takepos/public/auto_order.php?key=".dol_encode($row['rowid'])."'>".$urlwithroot."/takepos/public/auto_order.php?key=".dol_encode($row['rowid'])."</a>";
+		print "<a target='_blank' href='".$urlwithroot."/takepos/auto_order.php?key=".dol_encode($row['rowid'])."'>".$urlwithroot."/takepos/auto_order.php?key=".dol_encode($row['rowid'])."</a>";
```

Comment:
Why removing this.
For security purpose, every page that can be accessed without a login/pass must be stored into a directory "public".